source/includes/usage-examples/code-snippets/UpdateMany.java

```diff
@@ -0,0 +1,44 @@
+// ignored first line
```

Comment:
apparently this ignored first line behaviour is going to disappear soon, I can't find the platform ticket for it though.

Reply:
Release for this feature is tonight, so I'll create a separate PR to address all the issues here and other driver repos.

Reply:
nice, thanks for staying on top of that!